src/index.js

```diff
@@ -0,0 +1,21 @@
+const fetch = require("node-fetch");
+
+const readline = require('readline/promises').createInterface({
+  input: process.stdin,
+  output: process.stdout
+});
+
+readline.question('Would you like a dog or cat fact? ')
+  .then((response) => {
+    if (response.toLowerCase() === "cat") {
+      console.log("Too bad!");
```

Comment:
I feel like you're really leaving Emma out. Should we be better about this?

Reply:
I know... I know... can we just move forward anyways?